srtcore/queue.h

```diff
@@ -533,7 +532,7 @@ struct CMultiplexer
    CTimer* m_pTimer;        // The timer
 
    int m_iPort;         // The UDP port number of this multiplexer
-   int m_iIPversion;    // IP version
+   int m_iFamily;       // Address family (AF_INET or AF_INET6)
```

Comment:
Might be better to have the same name `m_iIPversion`, as used in other modules. Instead of `m_iFamily`